improver/calibration/rainforest_calibration.py

```diff
@@ -70,15 +71,22 @@ class ApplyRainForestsCalibration(PostProcessingPlugin):
     """
 
     def __new__(
-        cls, model_config_dict: Dict[str, Dict[str, Dict[str, str]]], threads: int = 1
+        cls,
+        model_config_dict: Dict[str, Dict[str, Dict[str, str]]],
+        threads: int = 1,
+        bin_data: bool = False,
     ):
+
         """Initialise class object based on package and model file availability.
 
         Args:
             model_config_dict:
                 Dictionary containing Rainforests model configuration variables.
             threads:
                 Number of threads to use during prediction with tree-model objects.
+            bin_data:
+                Bin data according to splits used in models. This speeds up prediction
+                if there are many data points which fall into the same bins for all models.
```

Comment:
bin_data:
                Bin data according to splits used in models. This speeds up prediction
                if there are many data points which fall into the same bins for all threshold models. Limits the calculation of common feature values by only calculating them once.